app/views/layouts/application.html.erb

```diff
@@ -104,15 +104,21 @@
       </div>
     <% end %>
     <%= yield %>
+    <% if FeatureFlag.enabled?(:runtime_banner) || ENV['E2E'] == 'true' %>
+      <div class="Browser-iOS">
+        <div class="runtime-banner">
+          <a href="<%= URL.deep_link(request.path) %>" target="_blank" class="flex items-center flex-1">
+            <%= inline_svg_tag("logo-forem-app.svg", aria: true, class: "crayons-icon crayons-icon--default", width: 32, height: 32, title: "Forem App icon") %>
+            <span class="ml-2">Open with the Forem app...</span>
+          </a>
+          <button type="button" class="runtime-banner__dismiss crayons-btn crayons-btn--ghost crayons-btn--icon crayons-btn--inverted crayons-btn--s pl-2">
+            <%= inline_svg_tag("x.svg", aria: true, class: "crayons-icon", title: "Dismiss the banner") %>
```

Comment:
```suggestion
            <%= inline_svg_tag("x.svg", aria: true, class: "crayons-icon", title: "Dismiss banner: Open with the Forem app") %>
```

suggestion (not-blocking) - just a thought but given screen reader users will often bring up a list of all buttons/links on a page, it might help if it was more descriptive about which banner this relates to